src/Plugin.php

```diff
@@ -234,6 +239,35 @@ public static function xmlrpc_methods($methods) {
     return $methods;
   }
 
+  /**
+   * @implements wp_head
+   */
+  public static function wp_head() {
+    $data = [
+      'ajaxurl' => admin_url('admin-ajax.php'),
+      'consent_version' => CORE_STANDARDS_COOKIE_CONSENT_VERSION,
+    ];
+    echo '<script>var core_standards = ' . json_encode($data) . ';</script>';
+  }
+
+  /**
+   * Logs cookie consent message.
+   */
+  public static function log_cookie_consent() {
+    $consent = $_POST['consent'] ?? [];
+    $data = [
+      'consent' => $consent,
+      'timestamp' => current_time('timestamp'),
+      'user_agent' => $_SERVER['HTTP_USER_AGENT'] ?? '',
+      'version' => CORE_STANDARDS_COOKIE_CONSENT_VERSION,
+      'placed_url' => wp_get_raw_referer(),
+      'user' => get_current_user_id(),
+      'id' => uniqid(rand(), TRUE),
```

Comment:
Major: This generates yet another ID that doesn't match the ID in the stored cookie.